styles/_mandara-insight.scss

```
@@ -0,0 +1,1461 @@
+}
+
+.insight-spinner {
+  width: 36px;
+  height: 36px;
+  border: 3px solid rgba(174, 129, 255, 0.2);
+  border-top-color: var(--color-accent);
+  border-radius: 50%;
+  animation: insightSpin 0.8s linear infinite;
+}
+
+@keyframes insightSpin {
```

Stylelint (17.6.0)

[error] Expected keyframe name "insightSpin" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)